spec/models/assignment_spec.rb

```diff
@@ -563,6 +636,71 @@
     end
   end
 
+context 'An Assignment' do
+  before :each do
+    @assignment = create(:assignment, group_name_autogenerated: false)
```

Comment:
This needs to be a group assignment for the test at line 674 to pass. You can make this change by passing `group_max: 2` when creating the assignment.